app/globals.css

```diff
@@ -93,8 +83,84 @@
   .primary-gradient {
     background: linear-gradient(90deg, rgb(67, 183, 254), rgb(79, 72, 230));
   }
+
+  .codeTextArea {
+    padding-left: 28px;
+    font-family: monospace;
+    line-height: 1.5;
+    font-size: 14px;
+    resize: none;
+  }
+  .editorLineNumbers {
+    font-family: monospace;
+    padding-left: 10px;
+    line-height: 1.5;
+    color: #565671;
+    font-size: 14px;
+  }
+
+  /* Hide scrollbar for Chrome, Safari and Opera */
+  .no-scrollbar::-webkit-scrollbar {
+    display: none;
+  }
+  /* Hide scrollbar for IE, Edge and Firefox */
+  .no-scrollbar {
+    -ms-overflow-style: none; /* IE and Edge */
+    scrollbar-width: none; /* Firefox */
+  }
 }
 
+/* Firefox */
+* {
+  scrollbar-width: thin;
+  scrollbar-color: #3a3d4f #252939;
+}
+
+/* Chrome, Edge, and Safari */
+*::-webkit-scrollbar {
+  width: 15px;
+}
+
+*::-webkit-scrollbar-track {
+  background: #252939;
+  border-radius: 5px;
+}
+
+*::-webkit-scrollbar-thumb {
+  background-color: #3a3d4f;
+  border-radius: 14px;
+  border: 3px solid #252939;
+}
+
+/* React Datepicker */
+
 .react-datepicker__view-calendar-icon {
   @apply bg-black-700 !flex items-center rounded;
 }
+
+/* TinyMCE */
```

Comment:
I tried to convert these to plain css styles like:
```
.tox-editor-header {
   background-color: !red;
 }
```
but, it wasn't working for me...